fuzzing/config/config.go

```diff
@@ -255,50 +272,73 @@ func (p *ProjectConfig) WriteToFile(path string) error {
 // Validate validates that the ProjectConfig meets certain requirements.
 // Returns an error if one occurs.
 func (p *ProjectConfig) Validate() error {
+	// Create logger instance if global logger is available
+	logger := logging.NewLogger(zerolog.Disabled)
+	if logging.GlobalLogger != nil {
+		logger = logging.GlobalLogger.NewSubLogger("module", "fuzzer config")
+	}
+
 	// Verify the worker count is a positive number.
 	if p.Fuzzing.Workers <= 0 {
-		return errors.New("project configuration must specify a positive number for the worker count")
+		return errors.New("worker count must be a positive number (update fuzzing.workers in the project config " +
+			"or use the --workers CLI flag)")
 	}
 
 	// Verify that the sequence length is a positive number
 	if p.Fuzzing.CallSequenceLength <= 0 {
-		return errors.New("project configuration must specify a positive number for the transaction sequence length")
+		return errors.New("call sequence length must be a positive number (update fuzzing.callSequenceLength in " +
+			"the project config or use the --seq-len CLI flag)")
 	}
 
 	// Verify the worker reset limit is a positive number
 	if p.Fuzzing.WorkerResetLimit <= 0 {
-		return errors.New("project configuration must specify a positive number for the worker reset limit")
+		return errors.New("worker reset limit must be a positive number (update fuzzing.workerResetLimit in the " +
+			"project config)")
+	}
+
+	// Verify timeout
+	if p.Fuzzing.Timeout < 0 {
+		return errors.New("timeout must be a positive number (update fuzzing.timeout in the project config or " +
+			"use the --timeout CLI flag)")
```

Comment:
There are a few edits to comments we made for bad configs where we tell users how to fix it. I think it was good we added specificity for which flag to update, but part of me thinks "we rename these things a good amount for now, I suspect we'll do that again and forget to rename these, it's more maintenance, do we want to explicitly reference them by proper name or just keep it somewhat descriptive without doing so?"

Let me know what you think, I won't impose, I'm fine with this either way.

Reply:
Ah yea good call, maybe we update this down-the-line?

Reply:
I think this is pretty valid - I will update to remove the specificity and we can close #181

Reply:
Resolved in https://github.com/crytic/medusa/commit/a2cab0f0ed9ddc46c9faa0cf72f570a06feaaa0e